sky/optimizer.py

```diff
@@ -195,46 +277,39 @@ def _remove_dummy_source_sink_nodes(dag: 'dag_lib.Dag'):
 
     @staticmethod
     def _get_egress_info(
-        parent: task_lib.Task,
-        parent_resources: resources_lib.Resources,
-        node: task_lib.Task,
-        resources: resources_lib.Resources,
+        parent_resources: resources_lib.Resources, node: task_lib.Task,
```

Comment:
```suggestion
        parent_resources: resources_lib.Resources,
```
do we still need this?